statsmodels/tsa/statespace/kalman_filter.py

```diff
@@ -1253,36 +1258,35 @@ def update_filter(self, kalman_filter):
                 # produce forecasts, but forecast errors and the forecast
                 # error covariance matrix will be zeros - make them nan to
                 # improve clarity of results.
-                if self.nmissing[t] == self.k_endog:
+                if self.nmissing[t] > 0:
```

Comment:
general, it's not change here

Except for nobs, we should use underline in n_missing, k_endog,    n_xxx, k_xxx. (naming convention)